force-app/main/default/classes/tasks/CollectionsTasks.cls

```diff
@@ -6,7 +6,13 @@ public class CollectionsTasks {
   public static Integer countZerosInBitList(
     List<Integer> listWithZerosAndUnits
   ) {
-    return 0;
+    integer countZeros = 0;
+    for (Integer element : listWithZerosAndUnits) {
+      if (element == 0) {
```

Comment:
Is it possible to do this without this `if`?

Reply:
added no IFs solution, see last commit